pkg/controllers/generic/import_syncer.go

```diff
@@ -176,14 +176,20 @@ var _ syncer.UpSyncer = &importer{}
 
 func (s *importer) SyncUp(ctx *synccontext.SyncContext, pObj client.Object) (ctrl.Result, error) {
 	// check if annotation is already present
-	if pObj.GetAnnotations() != nil &&
-		pObj.GetAnnotations()[translate.ControllerLabel] == s.Name() &&
-		!s.syncerOptions.IsClusterScopedCRD { // only delete pObj if its not cluster scoped
-		err := ctx.PhysicalClient.Delete(ctx.Context, pObj)
-		if err != nil && !kerrors.IsNotFound(err) {
-			return ctrl.Result{}, err
+	if pObj.GetAnnotations() != nil {
+		if pObj.GetAnnotations()[translate.ControllerLabel] == s.Name() &&
+			!s.syncerOptions.IsClusterScopedCRD { // only delete pObj if its not cluster scoped
+			err := ctx.PhysicalClient.Delete(ctx.Context, pObj)
+			if err != nil && !kerrors.IsNotFound(err) {
+				return ctrl.Result{}, err
+			}
+			return ctrl.Result{}, nil
+		}
+
+		// check if back sync is disabled eg. for service account token secrets
+		if pObj.GetAnnotations()[translate.SkipBacksyncInMultiNamespaceMode] == "true" {
+			return ctrl.Result{}, nil
 		}
```

Comment:
I think we could move this to `IsManaged()`, and the resource wouldn't even get put into reconcile queue. Wdyt?

Reply:
yea, I guess I'll just move it to `excludeObject` and then its called in `IsManaged`